auth0_flutter/example/ios/Tests/WebAuth/WebAuthLoginMethodHandlerTests.swift

```diff
@@ -174,6 +174,18 @@ extension WebAuthLoginHandlerTests {
         sut.handle(with: arguments(without: Argument.maxAge)) { _ in }
         XCTAssertNil(spy.maxAgeValue)
     }
+    
+    // MARK: safariViewController
+    
+    func testAddsSafariViewController() {
+        sut.handle(with: arguments(withKey: SafariViewController.key, value: [String: Any]())) { _ in }
+        XCTAssertNotNil(spy.provider)
+    }
+    
+    func testDoesNotAddSafariViewController() {
+        sut.handle(with: arguments(without: SafariViewController.key)) { _ in }
+        XCTAssertNil(spy.provider)
+    }
```

Comment:
I was looking for a way to also test that the presentation style was applied properly, but was difficult given the nature of how it's constructed. Would love to hear if it's possible to go into that detail though.

Reply:
I had a go at using an initializer for this, in https://github.com/auth0/auth0-flutter/pull/198/commits/8ac971d5b419caf45ef24af91348e2d10e21256b - let me know what you think.

Reply:
Oops, I deleted my first comment by mistake. I saw it below, under "[Widcket](https://github.com/Widcket) requested changes on behalf of" and thought it was duplicated (which wouldn't suprise me). My apologies.